Edytor/plansza.py

```diff
@@ -0,0 +1,164 @@
+from copy import deepcopy
+
+
+class Plansza:
+
+    def __init__(self, h, w, d=1):
+        self.h = h
+        self.w = w
+        self.d = d #dokładność
+        self.siatka = []
+        a = self.w * ['.']
+        for i in range(self.h):
+            self.siatka.append(list(a))
+
+    def __repr__(self):
+        return str(self.siatka)
+
+    def wielkosc_planszy(self):
+        px_h = self.h*self.d
+        px_w = self.w*self.d
+        return px_h, px_w
+
+    def punkt_na_pole(self, y, x):
+        h = int(y/self.d)
+        w = int(x/self.d)
+        return h, w
+
+    def pole_na_punkt(self, h, w):
+        y = (h*self.d + 0.5*self.d)
+        x = (w*self.d + 0.5*self.d)
+        return y, x
+
+    def wstawianie_obiektu(self,h,w,obiekt):
+        if self.siatka[h][w] == '.':
+            self.siatka[h][w] = obiekt
+            return True
+        elif self.siatka[h][w] != '.':
+            return False
+
+    def zapytanie_punkt(self, y, x):
+        h, w = self.punkt_na_pole(y, x)
+        if self.siatka[h][w] == '.':
+            return "Pole wolne"
+        else:
+            return self.siatka[h][w]
+
+    def zapytanie_pole(self, h, w):
+        if self.siatka[h][w] == '.':
+            return "Pole wolne"
+        else:
+            return self.siatka[h][w]
+
+    def czyszczenie_pola(self, h, w):
+        if self.siatka[h][w] == '.':
+            return "Pole wolne"
```

Comment:
Funkcja przy udanym czyszczeniu zwraca None (bo nie ma return), a przy nieudanym napis. W ogóle to wydaje mi się, że jak ktoś wywołuje czyszczenie to chce tylko aby pole było wyczyszczone - tzn nie interesuje go czy coś było czy nie. Może niczego nie zwracać?